pages/frequently-asked-questions/index.page.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState, useEffect } from 'react'
```

Comment:
* This page will go under the overview section and with a link in the sidebar.
pages/overview/faq

* Can we rename frequently-asked-questions by faq?

* This page is missing the sidebar. Please see [slug].page.tsx inside overview to understand how to show it.

* I'd love to provide a experience like to this page: https://graphql.org/faq/ With groups, collapsing expanding effect and each question will have a mardown heading to able to generate self-links so people can link the questions externaly.

Reply:
This is also done